docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -500,6 +579,37 @@ status = ml_train_layer_set_property(layers[1], "name=outputlayer",
 status = ml_train_model_add_layer(model, layers[1]);
 ```
 
+### Get a layer
+
+`ml_train_model_get_layer()` get a layer from the model with the given name:
+
+```c
+int status = ML_ERROR_NONE;
+
+ml_train_model_h model;
+ml_train_layer_h add_layer;
+ml_train_layer_h get_layer;
+
+status = ml_train_model_construct(&model);
+if (status != ML_ERROR_NONE) {
+  // handle error
+}
+
+status = ml_train_layer_create(&add_layer, ML_TRAIN_LAYER_TYPE_INPUT);
+if (status != ML_ERROR_NONE) {
+  // handle error
+}
+status = ml_train_layer_set_property(add_layer, "name=inputlayer", NULL);
+if (status != ML_ERROR_NONE) {
+  // handle error
+}
+status = ml_train_model_add_layer(model, add_layer);
+if (status != ML_ERROR_NONE) {
+  // handle error
+}
```

Comment:
How about giving the notification saying
"layer object owned by model. Do not delete it"

Reply:
okay